setup.bat

```diff
@@ -1,5 +1,7 @@
 cd rg_speech_to_text
-python3.8 -m venv venv
+python -m venv venv
 rem venv\Scripts\activate : activating venv breaks the bat file execution
 venv\Scripts\python -m pip install --upgrade pip setuptools
+venv\Scripts\python -m pip install torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
can this not be part of the requirements-win.txt ?

Reply:
pytorch has different install command for pip and for conda. The requirements.txt and requirements-win.txt should contain only modules which can be installed in both pip and conda, of course with pip.
Currently I must keep conda setup, because the audio drivers in PyPI is not working for the new windows release candidate. If I move this line from here to requirements-win.txt, I cannot reuse from conda.

The ML framework libraries anyway is a good practice to pre-install before adding the rest of the requirements. because some have cuda, some don't have.. that doesn't fit as a common requirements.